front/src/Api/Events/IframeEvent.ts

```diff
@@ -76,3 +78,46 @@ export interface IframeResponseEvent<T extends keyof IframeResponseEventMap> {
 export const isIframeResponseEventWrapper = (event: {
     type?: string;
 }): event is IframeResponseEvent<keyof IframeResponseEventMap> => typeof event.type === "string";
+
+
+/**
+ * List event types sent from an iFrame to WorkAdventure that expect a unique answer from WorkAdventure along the type for the answer from WorkAdventure to the iFrame
+ */
+export type IframeQueryMap = {
+    getState: {
```

Comment:
make sure all entries contain query and answer (maybe some more typesciprt wizardry)

Reply:
I'm not sure how to do it (but if you forget to add "query" or "answer", Typescript will heavily complain because a lot of functions are relying on those 2 entries to exist.